### PR TITLE
Uploading default dashboards only in Main organisation

### PR DESCRIPTION
**Pull request by rishubhjain**
tendrl-bug-id: Tendrl/monitoring-integration#133

### CODE REVIEW

File: tendrl/monitoring_integration/grafana/grafana_org_utils.py [hunk 1]
```diff
@@ -36,6 +36,25 @@ def create_org(org_name):
...
+
+    config = maps.NamedDict(NS.config.data)
+    if utils.port_open(config.grafana_port, config.grafana_host):
+        resp = get("http://{}:{}/api/orgs/name/"
+                      "{}".format(config.grafana_host,
+                                  config.grafana_port,
+                                  org_name),
+                   auth=config.credentials)
+        try:
+            return resp._content
```
Comment (r0h4n) [inline tendrl/monitoring_integration/grafana/grafana_org_utils.py]: there is no reason to use private variable of a third party library (which can change without notice)

Either use resp.text or resp.content everywhere in monitoring-integration

Comment (rishubhjain) [reply tendrl/monitoring_integration/grafana/grafana_org_utils.py]: @r0h4n https://pastebin.com/2EAvXShi this is a sample response sent by Grafana.
So to access the content of the response I have to use resp._content

Comment (r0h4n) [reply tendrl/monitoring_integration/grafana/grafana_org_utils.py]: You need to study python "properties" and the request library more. using private variables is NOT a good python practice

Use resp.content

More: https://github.com/requests/requests/blob/master/requests/models.py#L811

Comment (rishubhjain) [reply tendrl/monitoring_integration/grafana/grafana_org_utils.py]: @r0h4n  ack, will read the libraries, thanks
Will update the PR

**Comment by rishubhjain** [comment]
tendrl-bug-id: Tendrl/monitoring-integration#136

**Comment by rishubhjain** [comment]
@r0h4n I have made the changes in the same PR and attached another bug w.r.t using private variables.
@r0h4n Please review

**Pull request merged**